src/radio-tile/radio-tile.scss

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+@mixin iui-radio-tile {
+  cursor: pointer;
+  width: $iui-xl * 5;
```

Comment:
Not for this PR, but if we keep hardcoded width, we might need to have `small` size for these.